app/models/batch_context.rb

```diff
@@ -63,11 +62,6 @@ def file_manifest
                                   staging_location:)
   end
 
-  # this method would be better named manifest_type, but we are using the name content_metadata_creation in the UI
-  def content_md_creation
-    content_metadata_creation
-  end
-
```

Comment:
unnecessary indirection -- just reference our new name of `processing_configuration` everywhere it is needed